Traceback (most recent call last):
  File "/home/jcava/Molecular_Dynamics/Molecule_Dynamics_GAN/GAN_V12.3.2_Potential_Hierarchy/gan.py", line 269, in <module>
    t,output,_ = generator(batch_size,100)
  File "/home/jcava/.conda/envs/pytorch-geometric/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/jcava/Molecular_Dynamics/Molecule_Dynamics_GAN/GAN_V12.3.2_Potential_Hierarchy/gan.py", line 122, in forward
    z = torch.sigmoid(self.mlp3(z.view(batch_size,240)))
  File "/home/jcava/.conda/envs/pytorch-geometric/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/jcava/.conda/envs/pytorch-geometric/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 298, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/home/jcava/.conda/envs/pytorch-geometric/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 294, in _conv_forward
    return F.conv1d(input, weight, bias, self.stride,
RuntimeError: Expected 3-dimensional input for 3-dimensional weight [30, 6, 1], but got 2-dimensional input of size [128, 240] instead
